docs/handbook/glossary.md

```diff
@@ -212,9 +212,7 @@ The group of people contributing to Metagame. See [MetaFam in the MetaManifesto]
 
 ## Web3
 
-<!-- TODO -->
-
-<!---------------- ⭐️  X  ⭐️ ---------------->
+An ecosystem of decentralized and peer-to-peer protocols, applications, and technologies seeking solutions to the internet's current pitfalls such as centralization, lack of data privacy, and rampant advertising.
```

Comment:
When you added this definition you also removed (probably unintentionally the --- ⭐️  X  ⭐️  ---  marker/separator.
The likes highlighted in red on the left were removed.

Because your commits are all split, it requires a little bit more git fu to fix them while keeping them split (doable, just harder).

I'll show you how to squash all them into one and how to incorporate other changes that reviewers may request.
For example `DAO` could link to a wikipedia page, or similar.